ra/ra.go

```diff
@@ -2167,6 +2169,10 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 		if err != nil {
 			return nil, err
 		}
+		if len(authzIDs.Ids) == 0 {
+			// This should never happen.
```

Comment:
:rofl:

Reply:
If I don't whisper this incantation our fate will never be sealed.